### PR TITLE
HTML/CSS & JavaScript capstone project - Conference page

### PR DESCRIPTION
**Pull request by oliverSCZ**
**Milestone: First Capstone Project**
In my first capstone (project) as a Micronaut, I did the following:

- Created a GitHub repo entitled, 'capstone-project'.
- Created a new branch, 'capstone-project' from main in accordance with GitHub flow rules.
- Added all files required and followed the necessary instructions (.gitignore, .html, .css, js, test.md, README.md).
- Committed files (as required).
- Effectively set up a HTML & CSS & JS linter.
- Ensured that irrelevant files were ignored by Git.
- Documented my work professionally and added a descriptive README.md file to the project's repository using the template provided.
- I followed all the instructions for make this webpage.
- Opened this pull request as instructed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  <section class="partners">
+    <h3 class="our-partners">Our Partners</h3>
+    <hr class="underline">
+    <div class="logo-container">
+      <a class="sponsor-link" href="https://www.snapchat.com/" target="_blank" rel="noopener noreferrer"><i class="sponsor-icon fab fa-snapchat"></i></a>
+      <a class="sponsor-link" href="https://www.slack.com/" target="_blank" rel="noopener noreferrer"><i class="sponsor-icon fab fa-slack"></i></a>
+      <a class="sponsor-link" href="https://www.github.com/" target="_blank" rel="noopener noreferrer"><i class="sponsor-icon fab fa-github"></i></a>
+      <a class="sponsor-link" href="https://www.mozilla.org/" target="_blank" rel="noopener noreferrer"><i class="sponsor-icon fab fa-firefox-browser"></i></a>
+    </div>
+  </section>
```
Comment (oxenprogrammer) [inline index.html]: **Suggested Changes**
- [x] The `hr` underline is a little lower compared to the design, please move it a little closer to the `Partners` text.
- [x] The `icons` don't show on the hosted page as shown in the below screenshot. Also, please find the right icons or use text and style it as close as possible.
Yours
partner

Expected Design
partners

File: about.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          <ul class="navbar menu">
+  
+              <li><a href="index.html" class="menu-i">Home</a></li>
+              <li><a href="#services" class="menu-i">Our Services</a></li>
+              <li><a href="#contact" class="menu-i">Contact Us</a></li>
+              <li><button type="button" class="sign-up btn"> Sign Up</button></li>
+          </ul>
+          <div class="menu-btn">
+              <i class="fas fa-bars"></i>
+          </div>
```
Comment (oxenprogrammer) [inline about.html:43]: **Suggested Changes**
- [x] The mobile hamburger should be on the left side. Please let us move it.
- [x] It's background is transparent or the same color/pattern as the top section background. Currently yours is white background.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        <h4 class="exhib-section-title">${section.name}</h4>
+        <p class="exhib-section-brief">
+        ${section.brief}
+        </p>
+        <hr class="line-cards">
+        <p class="exhib-section-description">
+        ${section.description}
+        </p>
+      </div>
+    </li>`;
```
Comment (oxenprogrammer) [inline script.js:101]: **Optional Changes**
- [ ] We could try and use JavaScript to `dynamically` create these HTML5 elements. i.e `document.createElement()`

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @oliverSCZ, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: INVALID ⛔❌**
Hello @oliverSCZ 👋
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not pass all the linter checks.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: about.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+      <nav class="language-navbar">
+        <ul class="navMenu">
+          <li class="lang-nav-item"><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+          <li class="lang-nav-item"><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li class="lang-nav-item"><a class="top-link">English</a></li>
+          <li class="lang-nav-item"><a class="top-link">My Page</a></li>
+          <li class="lang-nav-item"><a class="top-link">Logout</a></li>
+        </ul>  
+      </nav>
+      <nav class="sticky">
```
Comment (rotimiazeez) [inline about.html:30]: - [ ] Please note that on the `about` page mobile screen, there is a little overflow to the right. Kindly fix that for a better user interface😉
screenshot (33)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  <section class="partners">
+    <h3 class="our-partners">Our Partners</h3>
+    <hr class="underline">
+    <div class="logo-container">
+      <a class="sponsor-link" href="https://www.google.com/" target="_blank" rel="noopener noreferrer">GOOGLE</a>
+      <a class="sponsor-link" href="https://www.asus.com/" target="_blank" rel="noopener noreferrer">ASUS</a>
+      <a class="sponsor-link" href="https://www.ebay.com/" target="_blank" rel="noopener noreferrer">EBAY</a>
+      <a class="sponsor-link" href="https://www.oracle.org/" target="_blank" rel="noopener noreferrer">ORACLE</a>
+    </div>
```
Comment (rotimiazeez) [inline index.html:112]: - [ ] Kindly make sure the partners' logos are well aligned at the centre of the page, as seen in the original design👌

footer-cap

**Review by rotimiazeez — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (rotimiazeez) [inline index.html]: - [x] Kindly make sure the partners' logos are well aligned at the centre of the page, as seen in the original design👌

footer-cap

File: about.html [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    <header>
+      <nav class="language-navbar">
+        <ul class="navMenu">
+          <li class="lang-nav-item"><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+          <li class="lang-nav-item"><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li class="lang-nav-item"><a class="top-link">English</a></li>
+          <li class="lang-nav-item"><a class="top-link">My Page</a></li>
+          <li class="lang-nav-item"><a class="top-link">Logout</a></li>
+        </ul>  
+      </nav>
```
Comment (rotimiazeez) [inline about.html:29]: - [x] Please note that on the `about` page mobile screen, there is a little overflow to the right. Kindly fix that for a better user interface😉
screenshot (33)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @oliverSCZ 👋, 

### Great work on making the changes requested by a previous reviewer 👏 

### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.🙄

 ## Required Changes ♻️
_Check the comments under the review._
> Kindly make the suggested changes and you are all good🚀🎉
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @oliverSCZ 😉, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oliverSCZ** [comment]
Thank you Mr. @rotimiazeez  and Mr. @emmanuellekamwa, thank you for your feedback.

**Pull request merged**